tests/Mock/Connection/TestBroker.php

```diff
@@ -21,6 +21,8 @@ class TestBroker extends RestClient
 
     protected $apiEndpoint;
 
+    protected $responseData = '{"message": "Some JSON Response Data", "id": "1234"}';
```

Comment:
Hi @DavidGarciaCat, Thanks for the review. I moved it to a property so I could extend the class and override the property to mock different response messages. Is there a better way to achieve this?